packages/sdk/python/human-protocol-sdk/human_protocol_sdk/storage.py

```diff
@@ -196,16 +198,18 @@ def upload_files(self, files: List[object], bucket: str) -> List[str]:
                 raise e
 
             data = artifact.encode("utf-8")
-            data_sha = hashlib.sha1(data).hexdigest()
-            key = f"s3{data_sha}.json"
+            hash = hashlib.sha1(data).hexdigest()
+            key = hash
```

Comment:
Why did you remove file extension?

Reply:
Simply because, to follow ts sdk behavior. It does not append anything to the hash, and just use hash as a key.